### PR TITLE
[AJ-781] don't show upload tsv when WDS isn't running

### PR DESCRIPTION
**Pull request by jordansafer**
PR to hide the Upload TSV button when WDS isn't yet running in the app, to avoid confusion. Once the WDS app is in a "RUNNING" state, the upload button will appear. Checking whether the instance exists is out of scope for this change.

Future thoughts:
- We can continue to improve messaging around WDS state
- ~~When WDS is provisioning, the proxyUrl is found. I think we should ignore WDS until provisioning is completed, since the user can't interact with WDS in a provisioning state. This also is creating confusing messaging. This is somewhat because we are using the proxy URL as a proxy (sorry!) for whether WDS is running or not. If we want to stick with this, we shouldn't populate the field until WDS is running.~~
- I noticed that proxyUrl doesn't get unset if WDS is shutdown. I don't think this is a major issue, since the values will be unset if the user refreshes the page.

https://broadworkbench.atlassian.net/browse/AJ-781

Testing:
- Created a new workspace and verified that the sidebar was no longer shown
- Called leo swagger to create a WDS and verified that sidebar was added

Before:
Screenshot 2023-01-18 at 1 24 37 PM

After:
Screenshot 2023-01-18 at 1 41 35 PM

### CODE REVIEW

**Review by davidangb — APPROVED** [review]
works for me, thank you. I defer to PM on whether or not to show the "Tables" section in the left nav with "Data tables are unavailable" when WDS isn't running, as opposed to showing nothing.

**Comment by bethsheets** [comment]
Following up on David's comment - I propose not showing "Tables" (what you have). In my opinion, the simpler screen helps focus the user on the message.

File: src/libs/ajax/data-table-providers/WdsDataTableProvider.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ const getRelationParts = (val: unknown): string[] => {
 export const getWdsUrl = apps => {
   // look explicitly for an app named 'cbas-wds-default'. If found, use it, even if it isn't running
   // this handles the case where the user has explicitly shut down the app
-  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === 'cbas-wds-default')
+  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === 'cbas-wds-default' && app.status === 'RUNNING')
```
Comment (calypsomatic) [inline src/libs/ajax/data-table-providers/WdsDataTableProvider.ts]: The appName should stay in concert with the name changes in https://github.com/DataBiosphere/terra-ui/pull/3673 .  Are you both editing this line?  Whoever merges second would need to watch out!

Comment (jordansafer) [reply src/libs/ajax/data-table-providers/WdsDataTableProvider.ts]: Agreed, one of us will have to rebase-

**Review by aaronkanzer — APPROVED** [review]

File: src/libs/ajax/data-table-providers/WdsDataTableProvider.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ const getRelationParts = (val: unknown): string[] => {
 export const getWdsUrl = apps => {
   // look explicitly for an app named 'wds-${app.workspaceId}'. If found, use it, even if it isn't running
   // this handles the case where the user has explicitly shut down the app
-  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === `wds-${app.workspaceId}`)
+  const namedApp = apps.filter(app => app.appType === 'CROMWELL' && app.appName === `wds-${app.workspaceId}` && app.status === 'RUNNING')
```
Comment (davidangb) [inline src/libs/ajax/data-table-providers/WdsDataTableProvider.ts:89]: nit: the code comment above is now incorrect

**Pull request merged**